src/PlaywrightSharp.Tests/BaseTests/PlaywrightSharpLoader.cs

```diff
@@ -11,12 +11,29 @@ public class PlaywrightSharpLoader
     {
         internal static SimpleServer Server { get; private set; }
         internal static SimpleServer HttpsServer { get; private set; }
+        internal static string ContentRoot { get; private set; }
 
         internal static async Task SetupAsync()
         {
-            var downloaderTask = TestConstants.GetNewBrowserType().CreateBrowserFetcher().DownloadAsync();
-            Server = SimpleServer.Create(TestConstants.Port, TestUtils.FindParentDirectory("PlaywrightSharp.TestServer"));
-            HttpsServer = SimpleServer.CreateHttps(TestConstants.HttpsPort, TestUtils.FindParentDirectory("PlaywrightSharp.TestServer"));
+            string host = !TestConstants.IsChromium ? TestConstants.BROWSER_CDN : null;

```

Comment:
Why do we need this?

Reply:
to download the patched version.
see https://github.com/kblok/playwright/blob/b8bb58e48b47200a6d4311b48a67132945609d21/browser_patches/tools/check_cdn.sh

Reply:
What would be the difference between that and this https://github.com/hardkoded/playwright-sharp/blob/master/src/PlaywrightSharp.Firefox/FirefoxBrowserType.cs#L273 ?

Reply:
not sure. they seem to be different but might contain the same archives
https://playwright2.blob.core.windows.net and https://playwright.azureedge.net

Reply:
I don't see where check_cdn.sh is being used in tests.